ecosystem/typescript/sdk_v2/src/crypto/asymmetric_crypto.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Deserializable, Deserializer, Serializable, Serializer } from "../bcs";
```

Comment:
not completely sure what this file is for? is it for later when we might have different schemas, each schema would extend those classes? is it always going to use bcs to serialize/deserialize?

Reply:
Yep, that is the idea. Each PublicKey and PrivateKey will extend from base classes and required to implement the Serialize/Deserialize methods, including (sign for PrivateKey, and verify for PublicKey)

> is it always going to use bcs to serialize/deserialize

Hmm this I am not sure. But if new serialization format is needed, we can modify the serializer/deserializer to accommodate that.